ons_alpha/topics/models.py

```diff
@@ -41,19 +44,81 @@ class TopicPage(BaseTopicPage):
     parent_page_types = ["topics.TopicSectionPage"]
     subpage_types = ["articles.ArticleSeriesPage", "bulletins.BulletinSeriesPage", "methodologies.MethodologyPage"]
     page_description = "A specific topic page. e.g. Public sector finance or Inflation and price indices"
+    summary = models.CharField(blank=True, max_length=255)
+    topic_page_nav = models.CharField(blank=True, max_length=255)
+
+    content_panels = BasePage.content_panels + [
+        FieldPanel("summary"),
+        FieldPanel("topic_page_nav"),
+    ]
+
+    edit_handler = TabbedInterface(
```

Comment:
Suggestion: This is duplicated from the parent, clearly because it needs access to the `.content_panels`. Perhaps this should be moved to a property on the parent class so it can retrieve the right values.

```python
from django.utils.functional import classproperty

@classproperty
def edit_handler(cls):
    return TabbedInterface(
        [
            ObjectList(cls.content_panels, heading="Content"),
            ObjectList(
                [FieldPanel("topic")],
                help_text="Select the topics that this page relates to.",
                heading="Taxonomy",
            ),
            ObjectList(cls.promote_panels, heading="Promote"),
        ]
    )
```